docs/contributing/release_process.md

```diff
@@ -70,3 +70,20 @@ to download the executable for their platform and run _Black_ without a
 The created binaries are attached/stored on the associated
 [GitHub Release](https://github.com/psf/black/releases) for download over _IPv4 only_
 (GitHub still does not have IPv6 access 😢).
+
+## Moving the `stable` tag
+
+_Black_ provides a stable tag for people who want to move along as _Black_ developers
+see a version become reliable. Here the _Black_ developers will move once the release
```

Comment:
This was hard to parse on first read. Not sure if this was the intended meaning though, so feel free to ignore or reword :)

```suggestion
_Black_ provides a stable tag for people who want to move along as _Black_ developers
deem the newest version reliable. Here the _Black_ developers will move once the release
```